Moderation_Matrix.md

```diff
@@ -0,0 +1,52 @@
+# Moderation Matrix
+
+## 1 Zap (WARNING) 
+Minor violations such as:
+- "Mini-modding"
+  - Please leave moderation action to the moderators, use ModMail to report rule breaking or if you see something you’re unsure about in the community
+- Asking about topics or projects unrelated to The Odin Project (Rules 11 and 12)
+-Mild unprofessionalism: see this guide https://www.bitrix24.com/about/blogs/communications/instant-messaging-and-group-chat-etiquette-in-a-professional-environme.php 
+  - Drug related conversation (Rule 7)
+  - Special attention to treating everyone with respect
+  - Mental health issues discussion (Rule 12)
+- A second infraction of the following:
+  - Short circuiting a teaching moment (Rule 10)
+  - Windows (Rule 14)
+    - Asking for help on or with
+    - Encouraging use of
```

Comment:
I think instead of sub-bullet points here, we should put it together in one cohesive point.
```suggestion
- Asking for help with Windows or encouraging the use of it
```